internal/config/config.go

```diff
@@ -122,8 +124,9 @@ func (c *Config) Load(ctx context.Context) error {
 
 // NewFabricClientForSDK returns a terraform sdkv2 plugin compatible
 // equinix-sdk-go/fabricv4 client to be used to access Fabric's V4 APIs
+// Deprecated: migrate to NewFabricClientForFramework instead
 func (c *Config) NewFabricClientForSDK(d *schema.ResourceData) *fabricv4.APIClient {
-	client := c.newFabricClient()
+	client := c.newFabricClient(context.WithValue(context.Background(), ctxKey("fabricSDKv2"), "no_oauth_reload"))
```

Comment:
I'd recommend waiting to make this change until you can update all of the functions calling `NewFabricClientForSDK` to pass in the correct context.  The background context is probably not the right one to latch on to.